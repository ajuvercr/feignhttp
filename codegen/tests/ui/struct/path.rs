use feignhttp_codegen::feign;

// error: metadata path not specified or must be the first
//    |  #[get(aaa = "/aaa")]
//    |        ^^^^^^^^^^^^

struct Http;

#[feign("http://xxx")]
impl Http {
    #[get(aaa = "/aaa")]
    pub async fn get() -> Result<String> {}
}

fn main() {}
