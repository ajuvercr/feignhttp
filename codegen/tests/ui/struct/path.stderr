error: metadata path not specified or must be the first
  --> tests/ui/struct/path.rs:11:11
   |
11 |     #[get(aaa = "/aaa")]
   |           ^^^^^^^^^^^^
